app/views/users/_form.html.erb

```diff
@@ -26,6 +26,7 @@
       <%= text_f f, :firstname %>
       <%= text_f f, :lastname %>
       <%= text_f f, :mail %>
+      <%= checkbox_f f, :disabled %>
```

Comment:
Inline label help describing the effect may be useful. "Disabled users can't login to the system and can't use API. User can be enabled later again. Disabling will not affect active user sessions."